File: steakfrye-cs50-2018-x-mario-less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    //ask user for integer until it is between 0 and 23
    do
    {
        height = get_int("Height: ");
    }
    while (height > 23 || height < 0);
    {
        //create height line by line
        for (int h = 0; h < height; h++)
        {

            //print spaces for each line, subtracting 1 each time
            for (int i = height - 1; i > h; i--)
            {
                printf(" ");
            }

            //print hashtags for each line, starting with 2, and adding 1 each time
            for (int k = 0; k < h + 2; k++)
            {
                printf("#");
            }
            //add lines
            printf("\n");
        }
    }
}

File: pset6/mario.py
#### creates a half pyramid of height specified by user ####

from cs50 import get_int

#prompt user for integer until they give one between 1 and 23
while True:
    height = get_int("Height: ")
    if height >= 0 and height < 24:
        break
#integer for the spaces
m = height - 1
#iterate through given height
for i in range(height):
    #print spaces
    for j in range(m):
        print(" ", end="")
    #print hashtags
    for k in range((i + 2)):
        print("#", end="")
    m -= 1
    #print next line
    print()